samples/nrf_desktop/src/events/config_event.c

```diff
@@ -4,9 +4,7 @@
  * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
  */
 
-#include <assert.h>
 #include <stdio.h>
-#include <misc/printk.h>
```

Comment:
I know it's not related, but why do we use `snprintf` and not `snprintk` for logging events?

Reply:
discussed offline